### PR TITLE
{ACS} Addition to --context description fixes #19779

### LINKED ISSUE #19779: az aks get-credentials does not respect --context when --admin flag is passed
```shell
 $ az version
{
  "azure-cli": "2.28.0",
  "azure-cli-core": "2.28.0",
  "azure-cli-telemetry": "1.0.6",
  "extensions": {}
}
```

## Issue:

When both the `--context` and `--admin` flags are set for `az aks get-credentials`, the `context` flag is not fully respected. Instead, a context gets created with an `-admin` suffix. This is undocumented behavior.

## Steps to reproduce

`az aks get-credentials --resource-group rg-foo--name cluster-foo --overwrite-existing --admin --context cluster-bar`

The above results in a `~/kube/config` file entry with `cluster-foo` under `clusters:`, and `cluster-foo-admin` under `contexts:`.

## Expectations:

When `--context` option is specified, the context name should reflect the value specified with `--context`. No strings should be appended to the context name.

### PR DESCRIPTION
**Pull request by navba-MSFT**
Addition to --context description. This fixes #19779

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change.
[Component Name 2] `az command b`: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

 fixes #19779

### CODE REVIEW

**Comment by yonzhan** [comment]
ACS

**Review by wangzelin007 — APPROVED** [review]

**Comment by navba-MSFT** [comment]
@wangzelin007 Thanks for approving this PR. Could you please merge this PR once the checks are successful ? Thanks in advance.

**Comment by wangzelin007** [comment]
> @wangzelin007 Zelin Wang FTE Thanks for approving this PR. Could you please merge this PR once the checks are successful ? Thanks in advance.

Need two reviewers approve before merge, don't worry.

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**
